### DISCUSSION TITLE
Error: No such option: -e

### DISCUSSION THREAD

**Discussion by iamtecheater** [blackhatethicalhacking/TerminatorZ#3] [General]
Hey all, getting this error and everything is installed 100%

[YOUR ARE USING TerminatorZ.sh] - (v1.0) CODED BY Chris 'SaintDruG' Abou-Chabké WITH ❤ FOR blackhatethicalhacking.com for Educational Purposes only!                                                                        
CHECKING IF YOU ARE CONNECTED TO THE INTERNET!                                                                
++++ CONNECTION FOUND, LET'S GO!                                                                              
┌─────────────────────────────────────┐                                                                       
│Entering the Matrix for 5 seconds in:│
└─────────────────────────────────────┘
┌─┐
│3│
└─┘
┌─┐
│2│
└─┘
┌─┐
│1│
└─┘
Enter the domain: 10 01 11 00 01 10
hackycorp.com                                                                                                 
Usage: httpx [OPTIONS] URL                                                                                    
                                                                                                              
Error: No such option: -e       !THIS IS THE ERROR!

![image](https://user-images.githubusercontent.com/68564467/219621131-6d37a899-3f0a-402d-906b-909f6405bad0.png)

**Comment by blackhatethicalhacking** [comment]
It was 6 am when I wanted to introduce an Enter the Matrix XD, so I moved that part towards the end, and now it will display exit the Matrix and will work, you can git pull & git stash and try again and you will be ok :)

The reason was it was also adding some characters in enter domain  :)

**Comment by kaeptenbalu** [comment]
hey just installed it today and getting the same error

thanks in advance
![Screenshot_2023-03-12_09-45-11](https://user-images.githubusercontent.com/16517472/224536904-d2abc255-101f-4b60-8c64-b86908d55a61.png)

**Comment by blackhatethicalhacking** [comment]
Did you update to the latest version?

Try first:

`git stash & git pull`

**Reply by kaeptenbalu** [reply]
still getting the same error

**Reply by blackhatethicalhacking** [reply]
Can you share a screenshot to see, if the error is still related to 'toilet' since the dependency has been removed from the latest update
